sashimi-webapp/src/components/file-manager/UserInputs.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="userInputs">
+    <div class="searchBar inline-block">
+      <input type="text" placeholder="Search..." @changed="searchInput">
+    </div>
+    <div class="toggle-view inline-block">
+      <button class="navbar-buttons">
+        <span>ICON</span>
+      </button>
+      |
+      <button class="navbar-buttons">
+        <span>List</span>
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: ['value'],
+};
+
+</script>
+
+<style lang="scss">
+@import 'src/assets/styles/variables.scss';
+
+.userInputs {
+  overflow: hidden;
+  text-align: center;
+  margin: 60px 0 30px 0;
+  position: relative;
+
+  input {
```

Comment:
`input` and `span` (line 49) tags shouldn't be styled directly here. A named class should be assigned instead.

Also, this `scss` is not scoped. Styling `input` and `span` directly in this way will affect the entire component and its child.